src/Mvc/Mvc.Core/src/ApplicationModels/InferParameterBindingInfoConvention.cs

```diff
@@ -155,4 +162,25 @@ private bool IsComplexTypeParameter(ParameterModel parameter)
 
         return metadata.IsComplexType;
     }
+
+    private bool IsOptionalParameter(ParameterModel parameter)
+    {
+        if (parameter.ParameterInfo.HasDefaultValue)
```

Comment:
This might throw which is why I'd asked about re-using `ParameterDefaultValue`. See https://github.com/dotnet/aspnetcore/blob/main/src/Shared/ParameterDefaultValue/ParameterDefaultValue.cs#L21-L32

Reply:
The runtime team already removed the try/catch since the issue was fixed. 

https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Extensions/ParameterDefaultValue/ParameterDefaultValue.netcoreapp.cs

The only scenario that it still there is if it is running on .NET Standard, that might not be the simplest thing to do.

https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Extensions/ParameterDefaultValue/ParameterDefaultValue.netstandard.cs

So, I will keep without it right now and keep eyes if someone open issues related to that.